mr/src/main/java/org/elasticsearch/hadoop/rest/InitializationUtils.java

```diff
@@ -291,7 +299,8 @@ public static void validateSettingsForWriting(Settings settings) {
                 throw new EsHadoopIllegalArgumentException("Cannot use TTL on index/update requests in ES 6.x and " +
                         "above. Please remove the [" + ConfigurationOptions.ES_MAPPING_TTL + "] setting.");
             }
-        } else {
+        }
```

Comment:
Nit: You can leave this formatting as is. Keeps the history cleaner, and the previous format is consistent with the project code style.

Reply:
Revert to one line